webview/src/experiments/components/table/Cell.tsx

```diff
@@ -63,21 +63,24 @@ export const FirstCell: React.FC<
       <div className={styles.innerCell}>
         <CellRowActions {...rowActionsProps} />
         <RowExpansionButton row={row} />
-        <span
-          className={styles.bullet}
-          style={{ color: bulletColor }}
-          {...clickAndEnterProps(toggleExperiment)}
+        <CellHintTooltip
+          tooltipContent={`${
+            bulletColor ? 'Remove from plots' : 'Add from plots'
```

Comment:
[Q] Can we go with `Unplot` / `Plot`? Otherwise `Add from plots` should be `Add to plots`.

Reply:
Related to the choice made for context menu options [here](https://github.com/iterative/vscode-dvc/pull/2388/files#diff-7f59061e48c6626575b99739398edf57257f7650eafb4af90f3a7792911fcf46R76).